wasm/src/main/java/com/dylibso/chicory/wasm/types/Instruction.java

```diff
@@ -6,19 +6,17 @@ public class Instruction {
     private final int address;
     private final OpCode opcode;
     private final long[] operands;
```

Comment:
As a follow up, we need to fix this, as exposing an array means it is not immutable. To avoid boxing, we could keep it as an array internally, but add accessors `operand0()` and `operand1()`. The `operands()` method would return a clone of the array.

Reply:
make sense, let's follow up :+1: